app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  #  :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
```

Comment:
- [ ] @Kaweesi-Matia, I can't sign up successfully into your app. No password confirmation information was sent to me even after a couple of numbers of trials.
- Screenshot:
![Capture](https://user-images.githubusercontent.com/63087491/218042246-880d89d1-93ea-4bfa-9fec-70153f9d9311.PNG)